src/settings_internal.js

```diff
@@ -174,10 +174,14 @@ var WASM_FUNCTIONS_THAT_ARE_NOT_NAME_MANGLED = ['setTempRet0', 'getTempRet0', 's
 // Internal: value of -flto argument (either full or thin)
 var LTO = 0;
 
+// Whether we may be accessing the address 2GB or higher. If so then we need
+// to be using unsigned pointers in JS.
+var CAN_ADDRESS_2GB = 0;
+
 // Whether to emit DWARF in a separate wasm file on the side (this is not called
 // "split" because there is already a DWARF concept by that name).
 // When DWARF is on the side, the main file has no DWARF info, while the side
 // file, ending in .debug.wasm, has the same wasm binary + all the debug
 // sections.
 // This has no effect if DWARF is not being emitted.
-var SEPARATE_DWARF = 0;
+var SEPARATE_DWARF = 0;
```

Comment:
Sam will get you on this :)

Reply:
Oh, isn't this what @sbc100  prefers - no extra newline at the end?

Reply:
https://github.com/emscripten-core/emscripten/pull/10478#discussion_r382681590 was a recent conversation around that

Reply:
Ah.. now I'm confused... but I'll remove this change :smile: